test/delegates_test.dart

```diff
@@ -31,4 +33,20 @@ void main() {
       (PathWrapper<AssetPathEntity> e) => e.path.name == 'Screenshots',
     );
   });
+
+  test('Select the correct TextDelegate for locale zh', () {
+    final delegate = assetPickerTextDelegateFromLocale(
+      const Locale.fromSubtags(languageCode: 'zh'),
+    );
+
+    expect(delegate.confirm, '确认');
+  });
+
+  test('Select the correct TextDelegate for locale zh-Hant', () {
+    final delegate = assetPickerTextDelegateFromLocale(
+      const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant'),
+    );
+
+    expect(delegate.confirm, '確認');
```

Comment:
Tests should not hard-code a variable that is declared in the code base. Consider `expect(delegate, equalsTo(xxx))`.